app_dart/lib/src/request_handlers/get_branches.dart

```diff
@@ -7,34 +7,20 @@ import 'dart:async';
 import 'package:meta/meta.dart';
 
 import '../datastore/cocoon_config.dart';
-import '../foundation/providers.dart';
-import '../foundation/typedefs.dart';
-import '../foundation/utils.dart';
 import '../request_handling/body.dart';
 import '../request_handling/request_handler.dart';
 
-/// Queries GitHub for the list of all available branches on
-/// [config.flutterSlug] repo, and returns list of branches
-/// that match pre-defined branch regular expressions.
 @immutable
 class GetBranches extends RequestHandler<Body> {
   const GetBranches(
-    Config config, {
-    @visibleForTesting
-        this.branchHttpClientProvider = Providers.freshHttpClient,
-    @visibleForTesting this.gitHubBackoffCalculator = twoSecondLinearBackoff,
-  })  : assert(branchHttpClientProvider != null),
-        assert(gitHubBackoffCalculator != null),
-        super(config: config);
-
-  final HttpClientProvider branchHttpClientProvider;
-  final GitHubBackoffCalculator gitHubBackoffCalculator;
+    Config config,
+  ) : super(config: config);
 
   @override
   Future<Body> get() async {
-    final List<String> branchList = await getBranchList(
-        config, branchHttpClientProvider, log, gitHubBackoffCalculator);
+    final String branches = await config.flutterBranches;
 
-    return Body.forJson(<String, List<String>>{'Branches': branchList});
+    return Body.forJson(
+        <String, List<String>>{'Branches': branches.split(',')});
```

Comment:
The config should be returning `List<String>` if that's how it is used everywhere else in the code. These split calls are littered through out the code when we could just do it once in the getter for this variable.

Reply:
Added wrapper.
//cc @godofredoc related to your comment: https://github.com/flutter/cocoon/pull/739/files#r405849586